apps/meteor/tests/end-to-end/api/livechat/contacts.ts

```diff
@@ -788,6 +794,24 @@ describe('LIVECHAT - contacts', () => {
 			expect(res.body.offset).to.be.an('number');
 		});
 
+		it('should return only known contacts by default', async () => {
```

Comment:
If it should return only known, wouldn't this case fail? 🤔 We are asserting it as `false`.

Reply:
It's kinda confusing 😢 . We're asserting that the property `unknown` is false since we want it to be known. An `unknown: true` contact, is someone that is not known by the workspace.